### PR TITLE
Use EDOT acronym

### PR DESCRIPTION
**Pull request by mdbirnstiehl**
Update the OTel docs to use the Elastic Distribution for OpenTelemetry (EDOT) acronym

Also updated the EDOT Overview diagram and moved the liabilities information on the guided and manual config pages.

### CODE REVIEW

**Review by colleenmcginnis — APPROVED** [review]
🦉

File: README.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
-# Elastic Distributions for OpenTelemetry
+# Elastic Distributions of OpenTelemetry
 
 [OpenTelemetry](https://opentelemetry.io/docs/) is a vendor-neutral observability framework for collecting, processing, and exporting telemetry data.
-The Elastic Distribution for OpenTelemetry Collector (Elastic OTel Collector) is a supported, drop-in replacement distribution of the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) made up of mostly upstream components.
-You can send your telemetry data to Elastic Observability using OpenTelemetry the following ways:
+The Elastic Distribution of OpenTelemetry (EDOT) Collector is a supported, drop-in replacement distribution of the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) made up of mostly upstream components.
```
Comment (mlunadia) [inline README.md:4]: This section goes immediately about the EDOT Collectior making it seem as if the EDOT collector is everything.

See the title is Elastic Distributions of OpenTelemetry which includes collector and language SDKs separately. See this [blog draft](https://docs.google.com/document/d/1ExB_L5_gcQbjyGtr9S_zT7s2trOz2CZGiyW1dHGthe0/edit) for details on the full ecosystem

File: docs/configure-upstream-collector.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 You can build and configure a [custom collector](https://opentelemetry.io/docs/collector/custom-collector/) or extend the [OpenTelemetry Collector Contrib ](https://github.com/open-telemetry/opentelemetry-collector-contrib) distribution to collect logs and metrics and send them to Elastic Observability.
 
-For a more seamless experience, use the Elastic Distribution for the OpenTelemetry collector.
-Refer to the [guided onboarding](guided-onboarding.md) docs or the [manual configuration](manual-configuration.md) docs for more on configuring the Elastic OTel collector.
+For a more seamless experience, use the Elastic Distribution for the OpenTelemetry Collector.
+Refer to the [guided onboarding](guided-onboarding.md) docs or the [manual configuration](manual-configuration.md) docs for more on configuring the Elastic Distribution for the OpenTelemetry Collector.
 
 ## Upstream collector configuration examples
```
Comment (mlunadia) [inline docs/configure-upstream-collector.md:8]: We want to provide users with configurations that they can copy and paste, maybe store them in this repo for their use.

File: docs/configure-upstream-collector.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 You can build and configure a [custom collector](https://opentelemetry.io/docs/collector/custom-collector/) or extend the [OpenTelemetry Collector Contrib ](https://github.com/open-telemetry/opentelemetry-collector-contrib) distribution to collect logs and metrics and send them to Elastic Observability.
 
-For a more seamless experience, use the Elastic Distribution for the OpenTelemetry collector.
-Refer to the [guided onboarding](guided-onboarding.md) docs or the [manual configuration](manual-configuration.md) docs for more on configuring the Elastic OTel collector.
+For a more seamless experience, use the Elastic Distribution for the OpenTelemetry Collector.
```
Comment (mlunadia) [inline docs/configure-upstream-collector.md:5]: We want to make this as detailed as possible. Lets elaborate more on how to use [ocb](https://opentelemetry.io/docs/collector/custom-collector/) and what components should be included and minimize sending the user to external links to get through to doing it.

We should send users to external links only if they want to go deeper or do something custom/advanced.

File: docs/guided-onboarding.md [hunk 4]
```diff
@@ -1,18 +1,17 @@
 # Collect logs and metrics using the guided onboarding
-The guided onboarding in Kibana or in a serverless Observability project walks you through collecting logs and metrics using the Elastic Distribution for OpenTelemetry Collector.
-To configure the Elastic OTel Collector manually, refer to the [manual configuration](manual-configuration.md) docs.
+The guided onboarding in Kibana or in a serverless Observability project walks you through collecting logs and metrics using the Elastic Distribution of OpenTelemetry (EDOT) Collector.
```
Comment (mlunadia) [inline docs/guided-onboarding.md:2]: >The guided onboarding in Kibana or in a serverless Observability project

Is this how we differentiate between ESS, Onprem and serverless? Never seen Kibana as the name being exclusive to non-serverless

File: docs/guided-onboarding.md [hunk 5]
```diff
@@ -1,18 +1,17 @@
...
 
 ## Before you begin
-The onboarding has the following requirements and limitations:
+The onboarding has the following requirements:
 
 - The **Admin** role or higher is required to onboard system logs and metrics. To learn more, refer to [Assign user roles and privileges](https://www.elastic.co/docs/current/serverless/general/assign-user-roles).
 - Root privileges on the host are required to run the OpenTelemetry collector used in this quickstart.
-- The Elastic OTel Collector only works on Kubernetes, Linux, and MacOS systems.
-- Refer to [Elastic OpenTelemetry Collector limitations](collector-limitations.md) for known limitations when using the Elastic OTel Collector.
+- The EDOT Collector only works on Kubernetes, Linux, and MacOS systems.
```
Comment (mlunadia) [inline docs/guided-onboarding.md:10]: >The EDOT Collector only works on Kubernetes, Linux, and MacOS systems.

The collector works on more platforms but at this moment we are providing out of the box deployment and configurations for these platforms

File: docs/manual-configuration.md [hunk 6]
```diff
@@ -1,31 +1,30 @@
-# Manually configure the Elastic Distribution for OpenTelemetry Collector
-Collecting logs and host metrics with the Elastic Distribution for OpenTelemetry Collector without using the [guided onboarding](guided-onboarding.md) requires some manual configuration.
+# Manually configure the Elastic Distribution of OpenTelemetry Collector
+Collecting logs and host metrics with the Elastic Distribution of OpenTelemetry (EDOT) Collector without using the [guided onboarding](guided-onboarding.md) requires some manual configuration.
```
Comment (mlunadia) [inline docs/manual-configuration.md:2]: manual steps, we are providing the configuration

File: docs/manual-configuration.md [hunk 7]
```diff
@@ -1,31 +1,30 @@
...
 
 - **Your Elasticsearch endpoint**: From the help menu in Kibana or your serverless Observability project, select **Connection details** and copy the **Elasticsearch endpoint**.
 - **API key**:
    - **Kibana:** From the help menu, select **Connection details** and select **Create and manage API keys**. From the **API keys** page, select **Create API key**. Give your API key a name, select **Create API key**, and copy the new API key.
    - **Serverless:** From the help menu, select **Connection details** and select the **API key** tab. Give your API key a name, select **Create API key**, and copy the new API key.
 
-Then manually configure the Elastic OTel Collector to collect logs and metrics on a MacOS or Linux system:
+Then manually configure the EDOT Collector to collect logs and metrics on a MacOS or Linux system:
 
 1. Download and extract the standalone Elastic Agent for your platform. For more on downloading and extracting a standalone Elastic Agent, refer to the first step in [Install standalone Elastic Agents](https://www.elastic.co/guide/en/fleet/current/install-standalone-elastic-agent.html).
```
Comment (mlunadia) [inline docs/manual-configuration.md:22]: I see we are mixing the use of EDOT collector and standalone Elastic Agent, this is a tricky one so I don't have a specific solution but just a heads up that we should come up with a good definition of when we call it EDOT Collector and when we call it Elastic Agent.

Maybe we explain this upfront and then only use invoke EDOT Collector?

**Review by mlunadia — CHANGES_REQUESTED** [review]

File: docs/configure-upstream-collector.md (hunk 3 above)
Comment (mdbirnstiehl) [reply docs/configure-upstream-collector.md:5]: This is in progress in [this issue](https://github.com/orgs/elastic/projects/649/views/43?pane=issue&itemId=73752307). I'll pull this content out of the issue into a new PR so we can finalize the details in GH. I don't think that should stop us from merging this PR though.

File: docs/configure-upstream-collector.md (hunk 2 above)
Comment (mdbirnstiehl) [reply docs/configure-upstream-collector.md:8]: Same as the previous comment.

File: docs/manual-configuration.md (hunk 7 above)
Comment (mdbirnstiehl) [reply docs/manual-configuration.md:22]: I think it makes sense to explain that the EDOT collector uses the Elastic Agent as a wrapper, or however we want to phrase this. I'm not sure we can only use EDOT collector here, as the user needs to go into the agent directory to modify the otel config, and the directory is named `elastic-agent-*`.

File: docs/configure-upstream-collector.md (hunk 2 above)
Comment (mlunadia) [reply docs/configure-upstream-collector.md:8]: > Elastic Distribution for the OpenTelemetry Collector.

of

**Pull request merged**